platform/src/Playground.js

```diff
@@ -340,6 +344,19 @@ function createPanelForDefinitionId(panel){
 
             break;
 
+            case "CompositePanel":
+
+                newPanel = new CompositePanel(newPanelId);
+                if (panel.childPanels) {
```

Comment:
Yes, but this also applies to other types of panels in the switch statement.

Reply:
Absolutely. I was just pointing out another refactoring opportunity. This one isn't perhaps so specifically driven by the introduction of the new type of Panel, which is why I feel it may not need to be fixed for this PR to be accepted.